### DISCUSSION TITLE
Is the Job N* of Threads per plot  or in total?

### DISCUSSION THREAD

**Discussion by luisperezcipo** [swar/Swar-Chia-Plot-Manager#200] [Q&A]
When configuring a job you can choose the number of threads and ram, is it the number per plot or for the whole process?

thank you

**Comment by infpico** [comment]
Did you find answer?

**Comment by brano1990** [comment]
I'm also interested when I set up 3 fibers and I have 3 plots, will 9 threads be used?

**Reply by infpico** [reply]
I have 24threads and when i set up 2, all are working anyway...

**Comment by myVictor** [comment]
有结果了吗？

**Reply by lonefighter** [reply]
作者给的范例内存是4000，这么看起来是单任务的，但是线程给的8，然后又说只有第一阶段会用到多于1个线程，这么看又像是给的整个任务的，迷糊了

**Comment by babanana009** [comment]
Definitely for 1 plot.
